userspace/libsinsp/event.cpp

```diff
@@ -1197,33 +1197,37 @@ Json::Value sinsp_evt::get_param_as_json(uint32_t id, OUT const char** resolved_
 			const struct ppm_name_value *flags = (const struct ppm_name_value *)m_info->params[id].info;
 			uint32_t initial_val = val;
 
-			while(flags != NULL && flags->name != NULL && flags->value != initial_val)
+			while(flags != NULL && flags->name != NULL)
```

Comment:
Avoid breaking once `flags->value != initial_val` because `exact_match` needs that.